conversionSetup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#Installs all necessary apps to convert pdf to adoc
+# install asciidoctor
+
+sudo apt-get install asciidoctor
+
+asciidoctor -v
```

Comment:
Are we actually using ascii doctor?

Reply:
I don't think we are. This was in the original conversion.sh before I broke it up into an installation script and a conversion script. I have not been able to test and verify that the script works without this line, so I am not confident in completely removing it yet.

Reply:
validate this and once validated I'll approve this PR

Reply:
I ran a check on this, we are not using asciidoctor.

Reply:
ok cool, remove it from the PR and we can approve this

Reply:
An update to the file should have been pushed. Can you confirm the update?